package.json

```diff
@@ -10,6 +10,7 @@
     "tdd": "jest --watch",
     "package-term": "yarn workspace @gravitational/teleterm package",
     "build-term": "yarn workspace @gravitational/teleterm build",
+    "build-native-deps-for-term": "yarn workspace @gravitational/teleterm build",
```

Comment:
Shouldn't it be `yarn workspace @gravitational/teleterm build-native-deps`?

Reply:
lmao thanks, just before pushing this PR I was like "It looks fine to me but I hope someone else will check this". 😶‍🌫️

Initially I added just `build-native-deps` in the package.json for teleterm. The readme also said "Remember to run `yarn build-native-deps`". But when I tried to actually follow the instructions from scratch, I realized that I'd have to be in `packages/teleterm` in order to run that command, hence last minute addition of `build-native-deps-for-term` which I copy pasted from `build-term` and forgot to change the last part.